src/Flow.jl

```diff
@@ -59,6 +59,15 @@ lowerBoundary!(r,u,Φ,ν,i,j,N,::Val{true}) = @loop (
     Φ[I] = ϕuP(j,CIj(j,CI(I,i),N[j]-2),CI(I,i),u,ϕ(i,CI(I,j),u)) -ν*∂(j,CI(I,i),u); r[I,i] += Φ[I]) over I ∈ slice(N,2,j,2)
 upperBoundary!(r,u,Φ,ν,i,j,N,::Val{true}) = @loop r[I-δ(j,I),i] -= Φ[CIj(j,I,2)] over I ∈ slice(N,N[j],j,2)
 
+
```

Comment:
Docs missing

Reply:
I am not sure why the change is included here. Maybe because I accidentally add a blank line here. This is the building blocks for Periodic BC

Reply:
I just highlighted the line on top of your new function, where docs should be including describing the function header and what it does.

Reply:
That functions isn't new to this PR. @TzuYaoHuang and/or @marinlauber should add docs for periodic BCs (including something on the main page mark-up) but not in this PR.

Reply:
Ah my bad.